src/ngraph/frontend/onnx_import/utils/broadcasting.cpp

```diff
@@ -17,12 +17,85 @@
 #include <numeric>
 #include <vector>
 
+#include "ngraph/op/broadcast.hpp"
+#include "ngraph/op/reshape.hpp"
+
 #include "broadcasting.hpp"
 
 namespace ngraph
 {
     namespace onnx_import
     {
+        std::vector<Shape> calculate_numpy_broadcast_shape(Shape left_shape, Shape right_shape)
```

Comment:
Please add header for `Shape` class.

Reply:
If this is only a helper function pleas make it `static` and outside `ngraph` namespaces scope.

Reply:
Pass arguments as const references and create function local copies where needed. You may name them `extended_left_shape` like.